tests/crontime.test.ts

```diff
@@ -778,3 +778,19 @@ describe('crontime', () => {
 		}).toThrow();
 	});
 });
+
+describe('validateCronExpression', () => {
+	it('should return true for valid cron expressions', () => {
+		expect(validateCronExpression('* * * * *').valid).toBe(true);
+		expect(validateCronExpression('0 0 * * *').valid).toBe(true);
+		expect(validateCronExpression('0 0 1 1 *').valid).toBe(true);
+		expect(validateCronExpression('*/5 * * * *').valid).toBe(true);
+	});
+
+	it('should return false for invalid cron expressions', () => {
```

Comment:
Please add assertions to make sure a `CronError` is present when `valid` is `false`.

Reply:
https://github.com/kelektiv/node-cron/pull/959/commits/adb4861de4b4cf969993351bf8bc33496a7cd26b done